### ISSUE TITLE
[Data@TAMU] Research projects community pages (All)

### ISSUE THREAD

**Issue by devangm** [TAMULib/data-dspace-angular#11]
**URL:** https://data-pre.library.tamu.edu/communities/e98002c6-8134-4424-90e5-59b721f6faca/search
**WCAG 2.1 AA compliant? :** No
**Violation:** 4.1.2 Name, Role, Value (A)
**Issues:**  Broken ARIA menu
**Notes:** There is a menubar with no items within it: `<div _ngcontent-dspace-angular-c2948902503="" role="menubar" class="dso-edit-menu d-flex">`
